inductiva/_cli/tasks.py

```diff
@@ -1,22 +1,71 @@
 """Tasks CLI subcommands."""
 from inductiva import tasks, utils
 from inductiva import _cli
+from inductiva.utils import format_utils
+from typing import Tuple
+from collections import defaultdict
 
 
-def list_tasks(args):
-    """List tasks."""
+def get_task_generic_info(list_of_tasks: list) -> Tuple[list, list]:
 
-    override_col_space = {
-        "Submitted": 18,
-        "Started": 18,
-        "Status": 10,
-        "Computation Time": 20,
-        "Resource Type": 20
+    table = defaultdict(list)
+
+    for task in list_of_tasks:
+        info = task.get_info()
+        status = task.get_status()
+
+        computation_end_time = info.get("computation_end_time", None)
+
+        execution_time = task.get_computation_time(fail_if_running=False)
+        if execution_time is not None:
+            if computation_end_time is None:
+                if status in ["started", "submitted"]:
+                    execution_time = f"*{execution_time}"
+                else:
+                    execution_time = "n/a"
+
+        executer = info["executer"]
+        if executer is None:
+            resource_type = None
+        else:
+            resource_type = executer["vm_type"]
+            if executer["n_mpi_hosts"] > 1:
+                resource_type += f" x{executer['n_mpi_hosts']}"
```

Comment:
Personal preference:

```suggestion
            n_mpi_hosts = executer["vm_type"]
            if n_mpi_hosts > 1:
                resource_type += f" x{n_mpi_hosts}"
```